recipes/bluesky/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "bluesky" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "e68dbcb64f98997a193408d1510c70d0ee00dedf79c35dad703f35d773f02e11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/NSLS-II/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
```

Comment:
A couple of issues here:

- `historydict` is missing py3.6 builds for at least osx and linux
- `super_state_machine` also missing py3.6 builds

Not sure why it looked like py3.5 was failing on CircleCI, but basically I went through and did `conda search <package>` looking for missing builds. Probably need to do the same for linux since I was defaulting to osx search. I'm guessing there is another package that's missing a py3.5 build on linux.

Reply:
Hmm ok, I thought I had gotten them all. Thank you very much!

Reply:
Is it possible to:
1. Get a better output in the CI? (This one gave no indication what is going wrong)
1. A command line tool to check if all the packages are available. (Load the recipe, create a list of all the build/run reqs, run `conda search ... package` in for loop)
1. A CI pre check which runs the aforementioned command (maybe adds its info to the linter output).

Reply:
Honestly I think this is either a `conda` or `conda-build` bug, @CJ-Wright. They should be providing this info if they can't create the environment. It may already be fixed in more recent versions or in development, but it wouldn't hurt to raise the issue. Might try `conda-build` first, but you may get directed to `conda` in the end.

Reply:
Anything to do with bad hints is all conda.  Please don't point people to conda-build for that.  This is a well-known issue, spread across many conda issues.  https://github.com/conda/conda/search?utf8=%E2%9C%93&q=unsatisfiable&type=Issues

Conda-build 3 tries to make sure that the run and test dependencies are available before build, and might tell you something more, but it's still ultimately conda that is giving you these hints (or lack thereof).

Reply:
Hmm ok. Out of curiosity, why are we running on `conda-build 2.18.1` rather than `conda-build 3.1.3`?

Reply:
> Anything to do with bad hints is all conda. Please don't point people to conda-build for that. This is a well-known issue, spread across many conda issues.

Ok. TBH am generally unclear on how `conda-build` interacts with `conda` at that level. Typically we get good hints when trying to construct a bad `conda` environment. So have not understood why we don't get good hints when `conda-build` constructs the `test` environment. If you say it is a `conda` issue, I'll take it at face value. Just pointing out that this is confusing even for someone from the community that is more familiar with these tools.

Is there an issue we should be adding this to? Don't want to be adding to the noise if you already know of one.

> Hmm ok. Out of curiosity, why are we running on conda-build 2.18.1 rather than conda-build 3.1.3?

It requires upgrading associated infrastructure as well. So it's not as simple as flipping a switch. ( https://github.com/conda-forge/conda-forge.github.io/issues/460 )

Reply:
It doesn't seem like that issue is going to get solved in the near future (they even have their own [label for it](https://github.com/conda/conda/issues?utf8=%E2%9C%93&q=label%3Atag%3Asolver-messaging+). Is there something that we can do in the meantime to at least help the issue?
I have created an issue for this https://github.com/conda/conda/issues/6453